src/DetailsView/components/load-assessment-dialog.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Tab } from 'common/itab';
+import { NamedFC } from 'common/react/named-fc';
+import { VersionedAssessmentData } from 'common/types/versioned-assessment-data';
+import { UrlParser } from 'common/url-parser';
+import {
+    ChangeAssessmentDialog,
+    ChangeAssessmentDialogProps,
+} from 'DetailsView/components/change-assessment-dialog';
+
+import * as React from 'react';
+import { DetailsViewActionMessageCreator } from '../actions/details-view-action-message-creator';
+
+export type LoadAssessmentDialogDeps = {
+    urlParser: UrlParser;
+    detailsViewActionMessageCreator: DetailsViewActionMessageCreator;
+};
+
+export interface LoadAssessmentDialogProps {
+    deps: LoadAssessmentDialogDeps;
+    prevTab: Tab;
+    newTab: Tab;
+    show: boolean;
+    loadedAssessmentData: VersionedAssessmentData;
+    tabId: number;
+    onClose: () => void;
+}
+
+export const LoadAssessmentDialog = NamedFC<LoadAssessmentDialogProps>('LoadAssessment', props => {
+    const loadAssessment = () => {
+        props.deps.detailsViewActionMessageCreator.loadAssessment(
+            props.loadedAssessmentData,
+            props.tabId,
+        );
+        props.onClose();
+    };
+
+    const continuePreviousAssessment = (event: React.MouseEvent<any, MouseEvent>) => {
+        props.deps.detailsViewActionMessageCreator.continuePreviousAssessment(event);
+        props.onClose();
+    };
+
+    const dialogProps: ChangeAssessmentDialogProps = {
+        deps: props.deps,
+        prevTab: props.prevTab,
+        newTab: props.newTab,
```

Comment:
Since this is just being passed through and is not used later, we can also remove this.

Reply:
this has been removed.